lib/s3_configuration.rb

```diff
@@ -24,6 +24,6 @@ def check!
   end
 
   def fake?
-    !configured? && Rails.env.development?
+    !configured? && !Rails.env.production?
```

Comment:
I find the double negative here a little hard to read. Would something like 

```
def real?
  configured? && Rails.env.production?
end
```

work in the context of the other changes you've made?

Reply:
I take your point, but that method ends up like this in the end:

```ruby
  def fake?
    !configured? && allow_fake?
  end
```

I think this is more readable and I think I prefer this to having to use `if !AssetManager.s3.real?` or `unless AssetManager.s3.real?` elsewhere in the codebase, e.g. `config/routes.rb`.